net/pfSense-pkg-Avahi/files/usr/local/pkg/avahi/avahi.inc

```diff
@@ -117,8 +117,29 @@ function avahi_sync_config() {
 		return;
 	}
 
-	log_error("Starting service avahi");
-	start_service('avahi');
+	/* Start/Restart Avahi, if a CARP VIP is set, check its status and act
+         * appropriately. */
```

Comment:
Indents must be tabs, not spaces (from here down there is a mix)

Reply:
fixed